### PR TITLE
Ask to select select which workspace to use for subscription

### PR DESCRIPTION
**Pull request by meln1k**
# Description

# To-Dos

- [x] Add test coverage for new or updated functionality
- [x] Lint and test with `nox`

# Code of Conduct

By submitting this pull request, I agree to follow the [code of conduct](https://some.engineering/code-of-conduct).

### CODE REVIEW

File: fixbackend/subscription/router.py [hunk 1]
```diff
@@ -54,13 +50,17 @@ async def aws_marketplace_fulfillment(
...
         fix_aws_marketplace_token: str = Cookie(None, alias="fix-aws-marketplace-token"),
     ) -> Response:
         if maybe_user is None:  # not logged in
             add_url = request.scope["router"].url_path_for(AddUrlName)
             return RedirectResponse(f"/auth/login?returnUrl={add_url}")
         elif (user := maybe_user) and fix_aws_marketplace_token is not None:  # logged in and token present
-            await market_place_handler().subscribed(user, fix_aws_marketplace_token)
+            subscription = await market_place_handler.subscribed(user, fix_aws_marketplace_token)
+            if subscription.workspace_id is None:  # no workspace yet
+                response = RedirectResponse(f"/assigh-subscription?id={subscription.id}")
```
Comment (aquamatthias) [inline fixbackend/subscription/router.py:62]:
```suggestion
                response = RedirectResponse(f"/assign-subscription?id={subscription.id}")
```

File: fixbackend/subscription/router.py [hunk 2]
```diff
@@ -54,13 +50,17 @@ async def aws_marketplace_fulfillment(
...
+        market_place_handler: AwsMarketplaceHandlerDependency,
         fix_aws_marketplace_token: str = Cookie(None, alias="fix-aws-marketplace-token"),
     ) -> Response:
         if maybe_user is None:  # not logged in
             add_url = request.scope["router"].url_path_for(AddUrlName)
             return RedirectResponse(f"/auth/login?returnUrl={add_url}")
         elif (user := maybe_user) and fix_aws_marketplace_token is not None:  # logged in and token present
-            await market_place_handler().subscribed(user, fix_aws_marketplace_token)
+            subscription = await market_place_handler.subscribed(user, fix_aws_marketplace_token)
+            if subscription.workspace_id is None:  # no workspace yet
```
Comment (aquamatthias) [inline fixbackend/subscription/router.py:61]: If the user only has one workspace - it should be assigned automatically.

Comment (meln1k) [reply fixbackend/subscription/router.py:61]: Already happening in the handler

**Review by aquamatthias — APPROVED** [review]
Thanks!

**Pull request merged**